packages/next-app/src/app/metadata/(components)/OwnedPoolsSidebarItems.tsx

```diff
@@ -91,25 +130,41 @@ function PoolCard({ isSelected, pool }: { isSelected: boolean; pool: Pool }) {
 
   const poolName =
     poolType === "Weighted" && tokens
-      ? tokens.map((obj) => obj.symbol).join("/")
+      ? tokens.map((obj) => obj.symbol).join(" / ")
       : name;
   const weights =
     poolType === "Weighted" && tokens
-      ? tokens.map((obj) => (Number(obj.weight) * 100).toFixed()).join("/")
+      ? tokens
+          .sort((a, b) => (a.weight < b.weight ? 1 : -1))
+          .map((obj) => (Number(obj.weight) * 100).toFixed())
+          .join(" / ")
       : null;
 
   return (
-    <>
-      <div className="flex items-center space-x-3 self-stretch">
-        <p className="text-lg font-bold text-slate12">{poolName}</p>
-        {weights && <Badge isSelected={isSelected}>{weights}</Badge>}
+    <div className="py-2">
+      <div className="flex items-center justify-between space-x-3 self-stretch">
+        <p
+          className={cn(
+            "tracking-tighter text-lg font-bold text-slate12 max-h-7 truncate text-left",
+            weights ? "max-w-[60%] " : "w-full"
+          )}
+        >
+          {poolName}
+        </p>
+        {weights && (
+          <div className="max-w-[40%]">
+            <Badge isSelected={isSelected} isTrackingTighter>
+              {weights}
```

Comment:
I'd suggest to truncate this as well since you can always click & see the pool anyways

Reply:
I think that if we truncate this information then its value is lost. I think we either show all the weights or we remove this badge

Reply:
fwiw people can always go back to the pool page. But I'd lean towards having the same info that we have on the page to search the pool in the pool page.

Reply:
so we should add this pool's information here. Is that it?
![image](https://github.com/bleu-studio/balancer-pool-metadata/assets/63298341/e1314b62-284f-42d1-8bfb-4c022892d63f)